common/config.go

```diff
@@ -17,6 +18,28 @@ const (
 
 	// OpConfigCMCADuration default duration for cert-manager issued service certificate
 	OpConfigCMCertDuration = "certManagerCertDuration"
+
+	// OpConfigLogLevel the level of logs to be written
+	OpConfigLogLevel = "logLevel"
+
+	// The allowed values for OpConfigLogLevel
+	logLevelWarning  = "warning"
+	logLevelInfo     = "info"
+	logLevelDebug    = "debug"
```

Comment:
On second thought about the levels, I think naming similar to [Liberty](https://openliberty.io/docs/latest/log-trace-configuration.html) would be better. Suggest `warning`, `info`, `fine`, `finer` and `finest`